pynamodb/attributes.py

```diff
@@ -961,6 +962,13 @@ def __init__(self, hash_key=False, range_key=False, null=None, default=None, att
                 raise ValueError("'of' must be subclass of MapAttribute")
             self.element_type = of
 
+    def remove(self, indexes=None):
```

Comment:
Apologies for not suggesting earlier, but having thought about this a bit it's probably better to have a separate `remove_indexes` function on `ListAttribute` rather than overloading the existing `remove`. What do you think?

Reply:
Well... while adding the indexes attribute, I was also thinking in same, separate the methods. I will do that then, it makes more sense.

Reply:
And also, instead of using:
```
class ListAttribute:

    def remove_indexes(self, *indexes):
        ....

Thread.notes.remove_indexes([1,2,...])
```

I add like this:
```
class ListAttribute:

    def remove_indexes(self, *indexes):
        ....

Thread.notes.remove_indexes(1, 2, ...)
```
What do you think?

Reply:
Makes sense to use varargs 👍